homeassistant/components/lamarzocco/binary_sensor.py

```diff
@@ -45,6 +45,13 @@ class LaMarzoccoBinarySensorEntityDescription(
         available_fn=lambda device: device.websocket_connected,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    LaMarzoccoBinarySensorEntityDescription(
+        key="backflush_enabled",
```

Comment:
Can this also be a switch entity to combine the button entity and the state?

Reply:
No. You can only enabled this, will be disabled automatically after 15 seconds by device